ibu-imager/installation_configuration_files/scripts/installation-configuration.sh

```diff
@@ -114,4 +114,36 @@ if [ -d ${EXTRA_MANIFESTS_PATH} ]; then
 fi
 
 rm -rf /opt/openshift
+
+restore_IBU() {
+    K8S_KUBECONFIG_RAW="$(oc config view --raw --minify -ojson)"
+    K8S_KUBECONFIG_CLUSTER="$(jq -r '.clusters[0].cluster' <<< $K8S_KUBECONFIG_RAW)"
+    K8S_KUBECONFIG_USER="$(jq -r '.users[0].user' <<< $K8S_KUBECONFIG_RAW)"
+
+    # Extract the server
+    K8S_API_ENDPOINT="$(jq -r '.server' <<< $K8S_KUBECONFIG_CLUSTER)"
+
+    # Cluster CA certs
+    K8S_API_CA_CERTS=$(jq -r '."certificate-authority-data" | @base64d' <<< $K8S_KUBECONFIG_CLUSTER)
+
+    # User credentials
+    K8S_API_CLIENT_CERT=$(jq -r '."client-certificate-data" | @base64d' <<< $K8S_KUBECONFIG_USER)
+    K8S_API_CLIENT_CERT_KEY=$(jq -r '."client-key-data" | @base64d' <<< $K8S_KUBECONFIG_USER)
```

Comment:
Let's move this to a load_kubeconfig function because it's a lot of noise for `restore_IBU`